apps/relayer/src/relayer.service.ts

```diff
@@ -1,15 +1,25 @@
+import { OdisUtils } from '@celo/contractkit'
+import { AuthSigner, ServiceContext } from '@celo/contractkit/lib/identity/odis/query'
+import { replenishQuota } from '@celo/phone-number-privacy-common/lib/test/utils'
```

Comment:
Is this the right replenishQuota? looks like a test util

Reply:
Yeah, in ODIS we used this method only for E2E testing. We can migrate it out of the `test` module since we're using it as part of the service here.